.github/dependabot.yml

```diff
@@ -0,0 +1,11 @@
+version: 2
+
+updates:
+- package-ecosystem: npm
+  directory: "/"
+  schedule:
+    interval: daily
+  open-pull-requests-limit: 10
+  commit-message:
+    prefix: chore
+    include: scope
```

Comment:
shall we maybe exclude those that are pinned? Just to make sure we don't update something that's not wanted

Reply:
We should pin all dev deps and not pin/update any dependencies.

I'll update the config for that